src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -200,20 +209,27 @@ const ProteinsListItemInfo = (props: Props) => {
       <div className={styles.proteinSummary}>
         <>
           <div className={styles.proteinSummaryTop}>
-            {uniprotXref && domainsLoadingState === LoadingState.SUCCESS && (
-              <div className={styles.interproUniprotWrapper}>
-                <ProteinExternalReference
-                  source={ExternalSource.INTERPRO}
-                  accessionId={uniprotXref.accession_id}
-                  name={uniprotXref.name}
-                />
-                <ProteinExternalReference
-                  source={ExternalSource.UNIPROT}
-                  accessionId={uniprotXref.accession_id}
-                  name={uniprotXref.name}
-                />
-              </div>
-            )}
+            {proteinXrefs?.length > 0 &&
```

Comment:
`proteinXrefs` are returned by `getProteinXrefs`, right? Which always returns an array, right? If so, there's no need for the question mark before the `.length`